preflight-packages/jetstack.io/pods/pods.rego

```diff
@@ -0,0 +1,188 @@
+package preflight.pods
```

Comment:
Should we still have the `preflight` prefix in the Rego package name?

Reply:
Do we need it? If not my vote would be to remove - but we still have it in our other policies... might be better to remove it all at once - over all packages.

Reply:
it is not needed, just drop it from the `package` statement in the rego and from the `root-query` in the policy manifest.

Reply:
Done!